resources/js/requests/components/RequestsListing.vue

```diff
@@ -339,7 +365,7 @@ export default {
             this.page +
             "&per_page=" +
```

Comment:
why is removed the "&" in this section?

Reply:
It wasn't removed.
Seems github is highlighting it in red 🤔

Reply:
very strange